docs/releaseNotes.yml

```diff
@@ -66,6 +66,16 @@ items:
         - title: "#4809"
           link: https://github.com/emissary-ingress/emissary/pull/4809
 
+      - title: Add starupProbe to emissary-apiext server
```

Comment:
should we add this in the chart changelog too?

Reply:
No because its not part of Helm, it is part of the crds and apiext yaml that is generated. This is why I put it here and not in the chart changelog.